### PR TITLE
deepl: Add version 2.5.0

### PR DESCRIPTION
**Pull request by alexchapar**
## What type of Apps related PR is this? (check all applicable)

- [x] ✨ Add Manifest
- [ ] ♻️ Refactor
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 🚩 Other

## Description
DeepL is a tools that can help translate text in differente language

The x86 version seems to be limited to 2.4.0, since its create some "automation" issue I removed it. It will require a modification if/when the x86 version issue of DeepL is resolved

## Related Tickets & Documents
Not related

### Add/Modify a description in the manifest?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Check URLs?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Update to the latest version?

- [ ] 👍 yes
- [x] 🙋 yes, but <del>I need help</del> _I had to force the x86 version to the prior version (2.4.0)_
- [ ] 🙅 no, because they aren't needed

### Update Hashes?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Format the json manifest?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Notes in the manifest?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Application Dependencies?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Application Suggestions?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### Pre-installation steps to perform?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

### Post-installation steps to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

### File persistence to perform?

- [ ] 👍 yes
- [ ] 🙋 yes, but I need help
- [x] 🙅 no, because they aren't needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?

:flushed:

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
[Your changes do not pass checks.](https://github.com/Ash258/Scoop-GithubActions/wiki/Pull-Request-Checks)
### deepl

- [x] Description
- [x] License
- [x] Hashes
- [ ] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### deepl

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

**Review by Bibz87-CG — APPROVED** [review]

**Pull request merged**
